plugins/jobs/src/js/types/Job.ts

```diff
@@ -24,16 +27,13 @@ import {
   JobStatusSchema
 } from "#PLUGINS/jobs/src/js/types/JobStatus";
 import { JobHistoryRun } from "#PLUGINS/jobs/src/js/types/JobHistoryRun";
```

Comment:
Why have you decided to go with `#PLUGINS` alias even for the plugin itself?

Reply:
because vscode auto importer does it like that :trollface:

Reply:
I think we want to improve our tooling that we are able to write something like `jobsplugin/types/JobHistoryRun` like rxjs does